### PR TITLE
Wordpress migrate v2

### PR DESCRIPTION
**Pull request by dreamer-nitj**
Migrate the wordpress community image to v2 framework

Test Plan:
[x] ran the stub/stub_coverage/harden/harden_coverage manually on ephemeral vm and it passed

### CODE REVIEW

File: community_images/wordpress/bitnami/image.yml [hunk 1]
```diff
@@ -19,3 +19,40 @@ disclaimer: |
   Trademarks: This software listing is packaged by RapidFort. The respective trademarks mentioned in the offering are owned by the respective companies, and use of them does not imply any affiliation or endorsement.
 docker_links:
   - "[`6`, `6-debian-11`, `6.0.1`, `6.0.1-debian-11-r2`, `latest` (6/debian-11/Dockerfile)](https://github.com/bitnami/bitnami-docker-wordpress/blob/6.0.1-debian-11-r2/6/debian-11/Dockerfile)"
+input_registry:
+  registry: docker.io
+  account: bitnami
+output_registry:
+  registry: docker.io
+  account: ankitrapidfort
```
Comment (dreamer-nitj) [inline community_images/wordpress/bitnami/image.yml:27]: will need to change this to rapidfort once the review changes are approved

File: community_images/wordpress/bitnami/image.yml [hunk 2]
```diff
@@ -19,3 +19,40 @@ disclaimer: |
...
+        tag: "image.tag"
+    override_file: "overrides.yml"
+  - type: docker_compose
+    script: dc_coverage.sh
+    compose_file: docker-compose.yml
+    image_keys:
+      wordpress:
+        repository: "WORDPRESS_IMAGE_REPOSITORY"
+        tag: "WORDPRESS_IMAGE_TAG"
+  # - type: docker
```
Comment (dreamer-nitj) [inline community_images/wordpress/bitnami/image.yml:49]: wordpress docker test requires mariadb docker container to be up first, is there a way to achieve that with this framework?

File: community_images/wordpress/bitnami/k8s_coverage.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+. "${SCRIPTPATH}"/selenium_helper.sh
+
+JSON_PARAMS="$1"
+
+NAMESPACE=$(jq -r '.namespace_name' < "$JSON_PARAMS")
+RELEASE_NAME=$(jq -r '.release_name' < "$JSON_PARAMS")
+
+WORDPRESS_SERVER="${RELEASE_NAME}"."${NAMESPACE}".svc.cluster.local
+
+test_selenium "${NAMESPACE}" "${WORDPRESS_SERVER}"
```
Comment (dreamer-nitj) [inline community_images/wordpress/bitnami/k8s_coverage.sh:19]: add the newline at the end

File: community_images/wordpress/bitnami/dc_coverage.sh [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -x
+set -e
+
+JSON_PARAMS="$1"
+
+JSON=$(cat "$JSON_PARAMS")
+
+echo "Json params for docker compose coverage = $JSON"
```
Comment (dreamer-nitj) [inline community_images/wordpress/bitnami/dc_coverage.sh:10]: add the newline at the end

File: community_images/wordpress/bitnami/k8s_coverage.sh (hunk 3 above)
Comment (dreamer-nitj) [reply community_images/wordpress/bitnami/k8s_coverage.sh:19]: self review

**Comment by codervinod** [comment]
Please add file to v2, generate files and then re-run tests

**Pull request merged**